src/classes/CW_BatchAudienceGroupRelationAssign.cls

```diff
@@ -0,0 +1,138 @@
+/**
+* File Name   :    CW_BatchAudienceGroupRelationAssign.cls
+* Description :    B-115203 - Target and Control group Naming Convention
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 13/05/2014 Alex Panka     Initial version
+* 0.2 10/07/2014 Amar Reddy     B-122759 : Process Relationship based on Group Name
+*
+*/
+
+global class CW_BatchAudienceGroupRelationAssign implements Database.Batchable<sObject>{
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.